popdat/Cargo.toml

```diff
@@ -7,12 +7,16 @@ edition = "2018"
 [dependencies]
 abstutil = { path = "../abstutil" }
 anyhow = "1.0.35"
+flatgeobuf = { version = "*" }
```

Comment:
I noticed this brought in lots of new dependencies. `cargo tree -i -p openssl` points to a [dep](https://github.com/michaelkirk/flatgeobuf/blob/4d70a8c5c774fdc39d5fc95a3c36af5803804c87/src/rust/Cargo.toml#L19) on reqwest. This'll prevent us from building in WASM because of the dependency on native SSL. I think we'll need a way to change the reqwest features that fgb uses to be something like `default-features=false, features=["blocking", "rustls-tls"] }`. I'm always a little confused by overriding transitive deps, but I think the proper way is to add a feature to fgb that in turn changes the dep on reqwest, instead of using `patch.crates-io`

Reply:
Ooph, I once again have mostly forgotten about the wasm use case. 

Allegedly reqwest has some support for the wasm target, but I'm a little nervous what I'll find. I'll look into it.

Reply:
Oh right, this maybe isn't straightforward -- in our [wasm file loader](https://github.com/dabreegster/abstreet/blob/0a25d92e5322ae09ed8c7d9fc444e7cff58f742d/map_gui/src/load.rs#L112), we use a call that doesn't even expose progress, let alone let you make range requests. If there's not something easy we can do, we can just rethink having `game` depend on `popdat`, and instead just generate a scenario through `importer`.